rj_gameplay/stp/role/assignment/naive.py

```diff
@@ -32,12 +32,16 @@ def get_sorted_requests(requests: assignment.FlatRoleRequests) -> SortedRequests
         role_id: assignment.RoleId
         request: role.RoleRequest
 
-        sorted_requests: SortedRequests = [
-            {} for _ in range(role.Priority.NUM_PRIORITIES)
-        ]
-
+        request_dict = {}
         for role_id, request in requests.items():
-            sorted_requests[int(request.priority)][role_id] = request
+            key = int(request.priority)
```

Comment:
Why change priority from float to int? Here 1.1 and 1.2 would be given the same priority, when we don't want that.